From 5271586b55536665016d871df91c783a31e25b8b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Eil=C3=ADs=20N=C3=AD=20Fhlannag=C3=A1in?=
 <pidge@toganlabs.com>
Date: Thu, 26 Nov 2020 18:30:24 +0000
Subject: [PATCH 05/38] arch-intel-centerton-pci-id.patch

Copyright (C) 2014 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

Add the PCI device ID for the Intel Centerton Integrated Legacy Block
(ILB).

This device ID is used by a number of drivers, including GPIO and
Multifunction Devices.
---
 include/linux/pci_ids.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index d93f4170..d35b1f41 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2484,6 +2484,7 @@
 #define PCI_DEVICE_ID_INTEL_MRST_SD2	0x084F
 #define PCI_DEVICE_ID_INTEL_I960	0x0960
 #define PCI_DEVICE_ID_INTEL_I960RM	0x0962
+#define PCI_DEVICE_ID_INTEL_CENTERTON_ILB	0x0c60
 #define PCI_DEVICE_ID_INTEL_8257X_SOL	0x1062
 #define PCI_DEVICE_ID_INTEL_82573E_SOL	0x1085
 #define PCI_DEVICE_ID_INTEL_82573L_SOL	0x108F
-- 
2.29.2

